Bottom: d962b497ba70cac12ec7147506247cb683d2c6ce
Top:    2a1f3cf8a3910e369bfa4c9318444585ce3942cd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0871915..33e5dbc 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1253,8 +1253,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
